draft-ietf-quic-transport.md

```diff
@@ -2091,7 +2091,8 @@ different local addresses, as discussed in {{connection-id}}.  For this to be
 effective endpoints need to ensure that connections IDs they provide cannot be
 linked by any other entity.
 
-Endpoints MAY move to a new connection ID at any time.
+At any time, endpoints MAY change the Destination Connection ID they send to a
+value that has not been used on another path.
```

Comment:
Much better.  Can this be "has not been used previously on this path or another path."?

Reply:
It's OK to reuse a connection ID that has been used on the same path.  Unless the goal is to avoid temporal linkability, but that's far less of a pressing issue, I think.

One thing I want to allow is alternating connection IDs, which your suggestion would prevent.

Reply:
I'm happy with this PR, but can you explain why you'd want that?

Reply:
I don't necessarily, but other have mentioned a desire to put multiple connection IDs on the same path.  This makes one connection look like multiple.

Reply:
Of course, you'd need your peer to do the same; otherwise, it's kind of pointless.

Reply:
I'd rather just have multiple connections or in v2, one could use multipath to do this.  Alternating between CIDs on a single path seems like a recipe for problems because few will exercise this option and it has dubious value in my opinion.

Reply:
If there is point to changing CID's on same path, then reordering can send packets with the old CID. I'm not sure there is, but it might have relevance to keep up with changes to router state. But I suppose a router will need to be able to track a CID forever if it keeps seeing it periodically.